spec/lex.dd

```diff
@@ -397,8 +397,9 @@ r"ab\n" // string is 4 characters,
 ---------------
 `hello`
 `c:\root\foo.exe`
-`ab\n`  // string is 4 characters,
-        // 'a', 'b', '\', 'n'
+`The "lazy" dog`
+`a"b\n`  // string is 5 characters,
+        // 'a', '"', 'b', '\', 'n'
 ---------------
```

Comment:
Or if you mean in the comment, then it would be inconsistent with the previous example:

https://github.com/GilesBathgate/dlang.org/blob/7fc048efeb02a282a1da280d818416c645ae2e0b/spec/lex.dd#L386